src/components/account/AccountSetting.jsx

```diff
@@ -34,25 +34,32 @@ const AccountSetting = () => {
   };
 
```

Comment:
I think I split the updateUserData axios calls in the utils file, not sure why the axios call was added back.